compliance/fedramp.md

```diff
@@ -0,0 +1,11 @@
+---
+description: fedramp
+keywords: standards, compliance, security, documentation
```

Comment:
I simply referenced some of the existing .md files that have those keywords. Will change as necessary.